requirements/lint.txt

```diff
@@ -1,13 +1,4 @@
 -r dev.txt
 
-# Security scanner
-bandit==1.7.0
-
-# Code checkers, linters and formatters
-black==21.6b0
-flake8==3.9.2
-flake8-blind-except==0.2.0
-flake8-debugger==4.0.0
-flake8-isort==4.0.0
```

Comment:
This one isn't in the flake8 additional-dependencies, because `isort` is already a separate check?

Reply:
Yes, exactly. `flake8-isort` depends on `isort` and just combine its capabilities with flake8 and that's something we do not need here.